lesson-4/4lesson_dz.py

```diff
@@ -0,0 +1,16 @@
+#1. Створити програму, яка буде очікувати введення тексту від користувача
+# і повідомляти — чи є введений текст “числом” чи “словом”.
+#2. Якщо введений текст “число”, програма має також вказати чи є воно парним чи непарним.
+#3. Якщо це “слово”, програма має вказати його довжину
+
+some_text = input('Enter some text')
+print(type(some_text))
+
+if some_text.isdigit():
+    num = int(some_text)%2
```

Comment:
It is not necessary to create a variable here. You can use `int(some_text) % 2` directly in condition: `if int(some_text) % 2:`

Reply:
done